### ISSUE TITLE
32bits failed tests

### ISSUE THREAD

**Issue by remicollet** [php-memcached-dev/php-memcached#318]
On x86_64, test suite passes

On i386:
```
TEST 35/81 [tests/gh_90.phpt]
========DIFF========
001+ Warning: mt_rand() expects parameter 2 to be integer, float given in /builddir/build/BUILD/php-pecl-memcached-3.0.1/NTS/tests/gh_90.php on line 10
========DONE========
FAIL Test for GH #90 [tests/gh_90.phpt] 

TEST 37/81 [tests/incrdecr.phpt]
========DIFF========
020+ int(1)
020- int(4294967297)
========DONE========
FAIL Memcached::increment() Memcached::decrement() [tests/incrdecr.phpt] 

TEST 38/81 [tests/incrdecr_bykey.phpt]
========DIFF========
017+ int(1)
017- int(4294967297)
========DONE========

Number of tests :   81                78
Tests skipped   :    3 (  3.7%) --------
Tests warned    :    1 (  1.2%) (  1.3%)
Tests failed    :    3 (  3.7%) (  3.8%)
Expected fail   :    0 (  0.0%) (  0.0%)
Tests passed    :   74 ( 91.4%) ( 94.9%)
```

Will try to propose a fix

**Comment by sodabrew** [comment]
Tagging this issue for 3.0.2 release. If it's preventing you from making distribution builds I'll be sure to cut the release as soon as we sort out the #319 #320 #321 PRs

**Comment by sodabrew** [comment]
Resolved by #319 to remove the mandatory 64-bit tests and #321 to split them out as separate 64-bit-only tests when 64-bits are available.
